godot-core/src/builtin/file.rs

```diff
@@ -0,0 +1,459 @@
+    pub fn get_line(&self) -> IoResult<GodotString> {
+        let val = self.fa.get_line();
+        self.check_error()?;
+        Ok(val)
+    }
+
+    /// Reads the next [Variant] value from file. If `allow_objects` is set to `true`, objects will be decoded
+    /// For details check the [Godot documentation](https://docs.godotengine.org/en/stable/classes/class_fileaccess.html)
+    pub fn get_var(&self, allow_objects: bool) -> IoResult<Variant> {
+        let val = self.fa.get_var_ex().allow_objects(allow_objects).done();
+        self.check_error()?;
+        Ok(val)
+    }
+
+    // Writes
+    pub fn store_8(&mut self, value: u8) -> IoResult<()> {
+        self.fa.store_8(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_16(&mut self, value: u16) -> IoResult<()> {
+        self.fa.store_16(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_32(&mut self, value: u32) -> IoResult<()> {
+        self.fa.store_32(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_64(&mut self, value: u64) -> IoResult<()> {
+        self.fa.store_64(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_double(&mut self, value: f64) -> IoResult<()> {
+        self.fa.store_double(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_float(&mut self, value: f32) -> IoResult<()> {
+        self.fa.store_float(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_real(&mut self, value: f32) -> IoResult<()> {
+        self.fa.store_real(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_string(&mut self, value: GodotString) -> IoResult<()> {
+        self.fa.store_string(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_pascal_string(&mut self, value: GodotString) -> IoResult<()> {
+        self.fa.store_pascal_string(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_line(&mut self, value: GodotString) -> IoResult<()> {
+        self.fa.store_line(value);
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_csv_line(
+        &mut self,
+        values: PackedStringArray,
+        delim: GodotString,
+    ) -> IoResult<()> {
+        self.fa.store_csv_line_ex(values).delim(delim).done();
+        self.check_error()?;
+        Ok(())
+    }
+
+    pub fn store_variant(&mut self, value: Variant, full_objects: bool) -> IoResult<()> {
+        self.fa
+            .store_var_ex(value)
+            .full_objects(full_objects)
+            .done();
+        self.check_error()?;
+        Ok(())
+    }
+
+    // Varia
+    pub fn set_big_endian(&mut self, value: bool) {
+        self.fa.set_big_endian(value);
+    }
+
+    pub fn is_big_endian(&self) -> bool {
+        self.fa.is_big_endian()
+    }
+
+    pub fn get_path(&self) -> GodotString {
+        self.fa.get_path()
+    }
```

Comment:
```suggestion
    pub fn path(&self) -> GodotString {
        self.fa.get_path()
    }
```

Same for `path_absolute` and `position`.